modin/backends/base/doc_utils.py

```diff
@@ -0,0 +1,685 @@
+        axis : {{0, 1}}
+            Axis to match indice along for 1D `other` (list or QueryCompiler that represents Series).
+            0 is for index, when 1 is for columns.
+        """,
+        "arithmetic": """
+        level : int or label
+            In case of MultiIndex match index values on the passed level.
+        axis : {{0, 1}}
+            Axis to match indice along for 1D `other` (list or QueryCompiler that represents Series).
+            0 is for index, when 1 is for columns.
+        fill_value : float or None
+            Value to fill missing elements in the result of frame allignment.
+        """,
+    }
+
+    verbose_substitution = (
+        f"other {sign} self" if self_on_right else f"self {sign} other"
+    )
+    params_substitution = extra_params.get(op_type, "")
+    return doc_qc_method(
+        template,
+        extra_params=params_substitution,
+        operation=operation,
+        verbose=verbose_substitution,
+    )
+
+
+def doc_reduce_agg(method, refer_to, params=None, extra_params=None):
+    """
+    Build decorator which adds docstring for the reduction method.
+
+    Parameters
+    ----------
+    method : str
+        The result of the method.
+    refer_to : str
+        Method name in ``modin.pandas.DataFrame`` module to refer to for
+        more information about parameters and output format.
+    params : str, optional
+        Method parameters in the NumPy docstyle format to substitute
+        to the docstring template.
+    extra_params : sequence of str, optional
+        Method parameter names to append to the docstring template. Parameter
+        type and description will be grabbed from ``extra_params_map`` (Please
+        refer to the source code of this function to explore the map).
+
+    Returns
+    -------
+    callable
+    """
+    template = """
+        Get the {method} for each column or row.
+        {params}
+        Returns
+        -------
+        BaseQueryCompiler
+            One-column QueryCompiler with index labels of the specified axis,
+            where each row contains the {method} for the corresponding
+            row or column.
+        """
+
+    if params is None:
+        params = """
+        axis : {{0, 1}}
+        level : None, default: None
+            Serves the compatibility purpose. Always has to be None.
+        numeric_only : bool, optional"""
+
+    extra_params_map = {
+        "skipna": """
+        skipna : bool, default: True""",
+        "min_count": """
+        min_count : int""",
+        "ddof": """
+        ddof : int""",
+        "*args": """
+        *args : iterable
+            Serves the compatibility purpose. Does not affect the result.""",
+        "**kwargs": """
+        **kwargs : dict
+            Serves the compatibility purpose. Does not affect the result.""",
+    }
+
+    params = "".join(
+        [params]
+        + [
+            extra_params_map.get(param, f"{param} : object\n")
```

Comment:
this template for missing arguments looks different, it doesn't include a newline or some whitespace in the beginning of the help line... something's off here.

Reply:
right, aligned this template to the ones in the `extra_params_map`